Makefile

```diff
@@ -13,6 +13,8 @@ build:
 	go build -o $(BUILD_ARCH)/$(BIN_DIR)/dp-filter-api cmd/$(MAIN)/main.go
 debug: build
 	HUMAN_LOG=1 go run -race cmd/$(MAIN)/main.go
+acceptance: build
```

Comment:
actually, I just noticed that this should probably not depend on `build` as it `run`s it directly

Reply:
I removed it, I also did this for `debug` as its the same command